### DISCUSSION TITLE
Searching in CMS pages.

### DISCUSSION THREAD

**Discussion by nul77** [breezefront/community#156] [1. Help] [answered]
Hi, 

We use the Breeze Evolution theme and added smile/elasticsuite and swissup/module-breeze-smile-elasticsuite.
This works fine for searching products & categories. Now I've installed the module smile/module-elasticsuite-cms-search to search for CMS pages.

Searchbox still works fine for categories & products, except if you search for a term that I know exists on an indexed cms page, it'll throw a JS-error:
`Cannot read properties of undefined (reading 'replace')`

How can I fix this? 

Thanks in advance, Tom

**Comment by vovayatsyuk** [comment, marked as answer]
Fixed: https://github.com/breezefront/module-breeze-smile-elasticsuite/commit/13e813282dd3d5d5927282e034dbc9d020a62196

**Reply by nul77** [reply]
Thanks!
